docs/src/featureToggle.js

```diff
@@ -9,5 +9,5 @@ module.exports = {
   enable_redirects: true, // related to new structure change
   enable_mui_base_scope: true, // will be enabled after the migration
   enable_system_scope: true, // will be enabled after the migration
-  enable_joy_scope: false,
+  enable_joy_scope: process.env.NODE_ENV === 'development' || process.env.STAGING === true,
```

Comment:
This would work as well, localhost is considered as staging.

```suggestion
enable_joy_scope: process.env.STAGING === true,
```